src/guide/comparison.md

```diff
@@ -55,17 +55,23 @@ With React's JSX, there are minor differences from HTML, such as using `classNam
 Take this example below:
 
 ``` jsx
-render () {
+render() {
   const { items } = this.props
 
+  let children
+  if (items.length > 0) {
+    children = items.map(item =>
+      <li key={item.id}>
+        {item.name}
+      </li>
+    )
```

Comment:
Typed it on iPhone so maybe I missed something but this paren should correspond to opening `items.map(`.